From 1c4907dbc89ef9d944ca792c7edb7559b7bc1f06 Mon Sep 17 00:00:00 2001
From: Sridevi Godithi <sgodithi@nvidia.com>
Date: Thu, 22 May 2025 09:14:44 -0700
Subject: [PATCH] Add file to keep track of CVE patches backported to vllm-src
 starting from 25.05.

This commit is a squashed version of 2 different commits

CVE patches backported to 0.8.4
fixing CVE number

(cherry picked from commit c4cbb86fc5acd4705c22de2f9417279db68958ad)
(cherry picked from commit 26f77d4653713d75f5bcf2683007e3b9bff2dc16)
(cherry picked from commit 200e97716e0a64904434f2f29a417c447748a39d)
---
 nvidia-cve-patch.txt | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 nvidia-cve-patch.txt

diff --git a/nvidia-cve-patch.txt b/nvidia-cve-patch.txt
new file mode 100644
index 000000000000..40ef0518ed08
--- /dev/null
+++ b/nvidia-cve-patch.txt
@@ -0,0 +1,2 @@
+25.5 CVE-2025-30202 GHSA-9f8f-2vmf-885j https://gitlab-master.nvidia.com/dl/vllm/vllm/-/commit/044b74cb61511e2c4524649b3fb728e95d31a8ec
+25.5 CVE-2025-32444 GHSA-hj4w-hm2g-p6w5 https://gitlab-master.nvidia.com/dl/vllm/vllm/-/commit/044b74cb61511e2c4524649b3fb728e95d31a8ec
\ No newline at end of file
